pkg/server/biz/utils/retry.go

```diff
@@ -0,0 +1,24 @@
+package utils
```

Comment:
This package can be used by other components, what about put it into `pkg/util` package.

Reply:
ok